lib/v2/hrbust/jwzx.js

```diff
@@ -0,0 +1,41 @@
+const got = require('@/utils/got');
+const utils = require('./utils');
+
+const typeMap = {
+    351: {
+        name: '名师风采',
+    },
+    353: {
+        name: '热点新闻',
+    },
+    354: {
+        name: '教务公告',
+    },
+    355: {
+        name: '教学新闻',
+    },
+};
+
+module.exports = async (ctx) => {
+    const page = ctx.params.page || '12';
+    const base = utils.columnIdBase(ctx.params.type) + '&pagingNumberPer=' + page;
+    const res = await got.get(base);
+    const info = utils.fetchAllArticle(res.data, utils.JWZXBASE);
+
+    const details = await Promise.all(info.map((e) => utils.detailPage(e.detailPage, ctx.cache)));
+
+    ctx.state.json = {
+        info,
+    };
+
+    ctx.state.data = {
+        title: '哈理工教务处' + typeMap[ctx.params.type || 354].name,
+        link: base,
+        item: info.map((e, i) => ({
+            title: e.title,
+            description: utils.renderDesc(details[i].desc),
+            pubDate: e.pubDate,
+            link: e.detailPage,
+        })),
```

Comment:
Please cache the whole returned result instead of `description` only which requires RSSHub to extract `title`, `pubDate`, and `link` again even if it's a cache-hit situation